README.md

```diff
@@ -6,12 +6,30 @@ The tool is set to recheck the library every day and download new books. Books w
 
 ## Extra Features
 
-enable `RENAME_CHAPTERS` to rename files from `Track - #.mp3` to `### <Book Title> - <Chapter Title>` as provided by libro.fm
+### Rename Chapters
+Enable `RENAME_CHAPTERS` to rename files from `Track - #.mp3` to `### <Book Title> - <Chapter Title>` as provided by libro.fm
+Additionally, if you enable `WRITE_TITLE_TAG`, each track's ID3 `title` field will be set to `### <Chapter Title>` as provided by libro.fm.
 
-enable `WRITE_TITLE_TAG` to update each track's ID3 `title` field to `### <Chapter Title>` as provided by libro.fm.
+### Convert to M4B
 
-bind a host port to `8080` to access the service's `/update` webhook. This allows you to manually force a refresh (ie: when you just purchased a book).
+M4B is an audio book file format that is supported by many players. By default, libro.fm on;y supports downloading "chapterized" MP3 files i.e. every chapter is its own file.
+That's not very convenient and also not supported by tons of players.
```

Comment:
Probably dont need the extra info on M4B, lets keep it short and assume the user knows about m4b

Reply:
Reworked and simplified